dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -451,3 +451,16 @@ contextPoolConfig=Context Pool Properties
 contextPoolConfig.desc=Properties of the context pool.
 
 contextPoolConfig$Ref=Context pool properties reference
+
+bindAuthMechanism=Bind Authentication Mechanism
+bindAuthMechanism.desc=The authentication mechanism for binding to the LDAP server when searching or modifying an LDAP entry.
+
+bindAuthMechanism.none=Anonymous bind to the directory service.
```

Comment:
none - don't need anything? o

Reply:
Do you mean I should say that using `none` doesn't require setting additional attributes? Or we don't need a none option?

Reply:
I swapped to `bindAuthMechanism.none=Anonymous bind to the directory service. No additional login attributes are required.`